homeassistant/components/denonavr/translations/en.json

```
@@ -23,6 +23,7 @@
                 "title": "Select the receiver that you wish to connect"
             },
             "user": {
+                "description": "By default this integration use a Telnet connection to your receiver to receive realtime updates. Note that this may interfere with the Denon AVR Remote app on your phone. You can turn this option off after setting up the integration by clicking the Configure button.",
```

Afaik telnet is not used by any app on the phone, but by some (uncommon) hardware extensions or maybe other tools similar to home-assistant (or a second home-assitant instance).
We should mention the telnet issue. A common problem might be running a live home-assistant instance and a dev instance at the same time.
```suggestion
                "description": "By default this integration use a Telnet connection to your receiver to receive realtime updates. Note that only one telnet connection to your receiver can be established. This may interfere with your currenty setup. You can turn this option off after setting up the integration by clicking the Configure button.",
```

---

Resolved in https://github.com/home-assistant/core/pull/85980/commits/cec869a425593ed9da022614e544c81a39e9f1ac